src/main/java/picocli/CommandLine.java

```diff
@@ -4332,6 +4340,9 @@ protected int hashCodeImpl() {
                 /** Sets the name of the option or positional parameter used in the usage help message, and returns this builder. */
                 public T paramLabel(String paramLabel)       { this.paramLabel = Assert.notNull(paramLabel, "paramLabel"); return self(); }
     
+                /** Sets the name of the option or positional parameter used in the usage help message, and returns this builder. */
```

Comment:
Just noticed I didn't update this javadoc entry...

Reply:
If you push additional commits to your branch, they will show up in the PR...